.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
+data_file = coverage/.coverage
```

Comment:
I cannot find this data_file, is it something everybody has to provide by themselves?

Reply:
This is somewhat separate from the rest of the PR: I added sensible defaults for the `coverage` tool. The `data_file` is the one that will be created when you run for example

```bash
coverage run /path/to/verdi -p test_profile devel tests && coverage html -i
```

I set it to be in a folder with all the additional report / annotation files that `coverage` can generate.

Long term I will try to add a coverage report stage to travis. VERY long term we might have it fail if it gets too low ;)